client/js/screens/EditProfile/EditProfile.js

```diff
@@ -2,19 +2,42 @@ import React from "react";
 import { Text, View, TouchableOpacity, ScrollView } from "react-native";
 import EditProfileAbout from "../../components/EditProfileAbout";
 import EditProfileContact from "../../components/EditProfileContact";
+import styles from "./styles";
 
-const EditProfile = ({ toggleTab, switchTab, user }) => (
+const EditProfile = ({ toggleTab, secondTabActive, user }) => (
```

Comment:
This should be a stateful component, seperate the stateful logic from the Query, and move it into this component.